<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="2882" started-at="2018-05-01T01:22:01Z" finished-at="2018-05-01T01:22:04Z">
    <groups>
      <group name="downloading">
        <method signature="clientTest.downloadsAllFilesInCategory()[pri:1, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="downloadsAllFilesInCategory" class="com.github.dionmcm.ncts.syndication.client.clientTest"/>
        <method signature="clientTest.downloadsLatestInCategory()[pri:2, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="downloadsLatestInCategory" class="com.github.dionmcm.ncts.syndication.client.clientTest"/>
        <method signature="clientTest.hashMismatchInSyndicationThrowsException()[pri:3, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="hashMismatchInSyndicationThrowsException" class="com.github.dionmcm.ncts.syndication.client.clientTest"/>
      </group> <!-- downloading -->
    </groups>
    <test name="Default test" duration-ms="2882" started-at="2018-05-01T01:22:01Z" finished-at="2018-05-01T01:22:04Z">
      <class name="com.github.dionmcm.ncts.syndication.client.clientTest">
        <test-method status="PASS" signature="setUpMockServer()[pri:0, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="setUpMockServer" is-config="true" duration-ms="2005" started-at="2018-05-01T01:22:01Z" finished-at="2018-05-01T01:22:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMockServer -->
        <test-method status="PASS" signature="downloadsAllFilesInCategory()[pri:1, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="downloadsAllFilesInCategory" duration-ms="447" started-at="2018-05-01T01:22:03Z" finished-at="2018-05-01T01:22:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- downloadsAllFilesInCategory -->
        <test-method status="PASS" signature="deleteAllFilesInClientFolder()[pri:0, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="deleteAllFilesInClientFolder" is-config="true" duration-ms="5" started-at="2018-05-01T01:22:03Z" finished-at="2018-05-01T01:22:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteAllFilesInClientFolder -->
        <test-method status="FAIL" signature="downloadsLatestInCategory()[pri:2, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="downloadsLatestInCategory" duration-ms="48" started-at="2018-05-01T01:22:03Z" finished-at="2018-05-01T01:22:03Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.github.dionmcm.ncts.syndication.client.clientTest.downloadsLatestInCategory(clientTest.java:94)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- downloadsLatestInCategory -->
        <test-method status="PASS" signature="deleteAllFilesInClientFolder()[pri:0, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="deleteAllFilesInClientFolder" is-config="true" duration-ms="0" started-at="2018-05-01T01:22:03Z" finished-at="2018-05-01T01:22:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteAllFilesInClientFolder -->
        <test-method status="PASS" signature="hashMismatchInSyndicationThrowsException()[pri:3, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="hashMismatchInSyndicationThrowsException" duration-ms="46" started-at="2018-05-01T01:22:03Z" finished-at="2018-05-01T01:22:03Z">
          <exception class="com.github.dionmcm.ncts.syndication.client.HashValidationFailureException">
            <message>
              <![CDATA[File /Users/patrick/git/ncts-syndication-client/ncts-syndication-client-library/src/test/java/com/github/dionmcm/ncts/syndication/client/Test_Client/green1.r2 with hash 63640264849a87c90356129d99ea165e37aa5fabc1fea46906df1a7ca50db492 and length 11 once downloaded did not match the advertised hash thisHashIsIncorrect and/or length 11. Downloaded file deleted, process aborted.]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.github.dionmcm.ncts.syndication.client.HashValidationFailureException: File /Users/patrick/git/ncts-syndication-client/ncts-syndication-client-library/src/test/java/com/github/dionmcm/ncts/syndication/client/Test_Client/green1.r2 with hash 63640264849a87c90356129d99ea165e37aa5fabc1fea46906df1a7ca50db492 and length 11 once downloaded did not match the advertised hash thisHashIsIncorrect and/or length 11. Downloaded file deleted, process aborted.
at com.github.dionmcm.ncts.syndication.client.NctsFileDownloader.downloadFile(NctsFileDownloader.java:141)
at com.github.dionmcm.ncts.syndication.client.NctsFileDownloader.downloadEntry(NctsFileDownloader.java:111)
at com.github.dionmcm.ncts.syndication.client.SyndicationClient.download(SyndicationClient.java:96)
at com.github.dionmcm.ncts.syndication.client.SyndicationClient.downloadLatestFromCategories(SyndicationClient.java:121)
at com.github.dionmcm.ncts.syndication.client.SyndicationClient.downloadLatest(SyndicationClient.java:149)
at com.github.dionmcm.ncts.syndication.client.clientTest.hashMismatchInSyndicationThrowsException(clientTest.java:107)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- com.github.dionmcm.ncts.syndication.client.HashValidationFailureException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- hashMismatchInSyndicationThrowsException -->
        <test-method status="PASS" signature="deleteAllFilesInClientFolder()[pri:0, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="deleteAllFilesInClientFolder" is-config="true" duration-ms="1" started-at="2018-05-01T01:22:03Z" finished-at="2018-05-01T01:22:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteAllFilesInClientFolder -->
        <test-method status="PASS" signature="tearDownMockServer()[pri:0, instance:com.github.dionmcm.ncts.syndication.client.clientTest@498d318c]" name="tearDownMockServer" is-config="true" duration-ms="306" started-at="2018-05-01T01:22:03Z" finished-at="2018-05-01T01:22:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMockServer -->
      </class> <!-- com.github.dionmcm.ncts.syndication.client.clientTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
